### PR TITLE
Add cumulative mode to resolved incident plot

### PR DESCRIPTION
**Pull request by nathanathan**
Pivotal Story: https://www.pivotaltracker.com/story/show/146605873

### CODE REVIEW

**Comment by jgoley** [comment]
Looks like there is a chance the graph can be empty. I wonder if we should disable the button or somehow indicate there isn't enough data to show the plot?

Looks like the buttons break weirdly at smaller screen widths/resolutions. I'll work on this in the morning.

File: client/controllers/resolvedIncidentsPlot.coffee [hunk 1]
```diff
@@ -153,9 +194,19 @@ Template.resolvedIncidentsPlot.helpers
 Template.resolvedIncidentsPlot.events
   "click .incident-type-selector .cases": (e, instance)->
```
Comment (jgoley) [inline client/controllers/resolvedIncidentsPlot.coffee]: We should probably use `event, instance` as the parameters to stay consistent?

**Comment by jgoley** [comment]
@nathanathan It looks like the tests are failing because of a property on the incident which isn't  allowed by the schema. It appears to fail in master and in #386.

**Comment by nathanathan** [comment]
@jgoley Thanks for looking into the test failure. I think it was caused by some properties I added to the geoname lookup api responses for #385

**Pull request merged**